bar.go

```diff
@@ -268,13 +268,15 @@ func (b *Bar) SetPriority(priority int) {
 // if bar is already in complete state. If drop is true bar will be
 // removed as well.
 func (b *Bar) Abort(drop bool) {
+	if drop {
+		b.container.dropBar(b) // It is safe to call this multiple times with the same bar
+	}
```

Comment:
This one introduces undesired side effect, making `Abort(true)` always drop the bar even after completion. I think it will be unexpected for user given what doc comment states.